eventsource/request.py

```diff
@@ -2,7 +2,11 @@
 import argparse
```

Comment:
don't forget to (at least) add `from __future__ import print_function`. I'd add as well `unicode_literals`.